knowledge-base/how-to-load-telerik-report-viewer-resources-locally-in-angular-application-and-use-them-to-customize-the-viewer.md

```diff
@@ -0,0 +1,109 @@
+---
+title: How to load Telerik Report viewer resources locally in Angular application and use them to customize the viewer
+description: In some cases a modification of the default behavior, layout or styles of the HTML5 Report Viewer in an Angualr application is needed. This article elaborates further on how to load the required default resources locally.
+type: how-to
+page_title: How to load Telerik Report viewer resources locally in Angular application and use them to customize the viewer
+slug: how-to-load-telerik-report-viewer-resources-locally-in-angular-application-and-use-them-to-customize-the-viewer
+position: 
+tags: AngularReportViewer
+ticketid:
+res_type: kb
+---
+
+## Environment
+<table>
+    <tbody>
+	    <tr>
+	    	<td>Product Version</td>
+	    	<td>13.0.19.222</td>
+	    </tr>
+	    <tr>
+	    	<td>Product</td>
+	    	<td>Progress® Telerik® Reporting</td>
+	    </tr>
+	    <tr>
+	    	<td>Project Type</td>
+	    	<td>Angular application</td>
+	    </tr>
+    </tbody>
+</table>
+
+
+## Description
+Do you want to customize the report viewer's default behavior? Let's find out how.
+
+## Prerequisites
+Angular requires folder called assets to be used for serving custom files, so they could be included in the production build. This is done via the assets property in the .angular-cli.json file (or .angular.json for Angular 6+ applications).
+
+```JSON
+"assets": [ 
+    "src/assets",
+    ... 
+],
+```
+
+Check out the documentation for this feature here to learn how to use it: [https://github.com/angular/angular-cli/wiki/stories-asset-configuration](https://github.com/angular/angular-cli/wiki/stories-asset-configuration).
+
+## Important notes
+Keep in mind that this will require to merge the new changes for each of the local files manually on upgrade.
+
+## Solution
+The following three scenarios could be applied together (if all resources have to be modified). However, if you intend to modify just one resource, for example the script file, check only the corresponding section.
+
+### Scenario 1
+How to load viewer's script locally **(i.e. telerikReportViewer-<VERSION>.js)**?
+
+### Example
```

Comment:
It may look better if the Example/How to sections are smaller (i.e. ####) than the Scenario sections - just to look like sub-sections

Reply:
done